internal/debug/debug.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debug // import "go.opentelemetry.io/otel/internal/debug"
```

Comment:
Just to be clear you are looking to move these methods to `/internal/`?  That would mean when we call these within our code we have `internal.Info(...)`, `internal.Error(err,...)` and `internal.Debug(...)`.

Reply:
I was talking about the [internal/global](https://github.com/open-telemetry/opentelemetry-go/tree/main/internal/global) package. I think flattening into that package (and we call `global.Info(...)`, `global.Error(err, ...)`, `global.Debug(...)`) would work, but it might also make sense to just have it become a subpackage as well.